components/CreateCollectiveMiniForm.js

```diff
@@ -100,7 +100,7 @@ const msg = defineMessages({
   },
   examples: {
     id: 'examples',
-    defaultMessage: 'e.g. {examples}',
+    defaultMessage: 'e.g., {examples}',
```

Comment:
I expect this to be solved in another PR but this is currently broken on `main`.